lib/trento_web/controllers/fallback_controller.ex

```diff
@@ -75,6 +75,13 @@ defmodule TrentoWeb.FallbackController do
     |> render(:"422", changeset: changeset)
   end
 
+  def call(conn, {:error, :associated_database_not_found}) do
+    conn
+    |> put_status(:accepted)
+    |> put_view(json: ErrorJSON)
```

Comment:
`ErrorJSON`?
I think we can return an empty json with `json(%{})` or `even just try to `send_resp(conn, 202, %{})`.
Because I don't think we have a 202 view to render (unless phoenix has it by default)

Reply:
FIxed in https://github.com/trento-project/web/pull/3074/commits/b4b35325eb66fa446bfe6d312c82152e5bca72ac